doc/schema/schema.md

```diff
@@ -79,7 +90,7 @@ Observed data on which the `posterior` is conditional. It should only contain da
 Model constants, data included in the model which is not modeled as a random variable. It should be the data used to generate the `posterior` and `posterior_predictive` samples.
```

Comment:
Wouldn't these also be used to generate samples from the prior definition described immediately below?